index.html

```diff
@@ -280,16 +280,15 @@ <h2>Keep in touch</h2>
     </div>
   </div>
 
-  <div class="footer">
+  <div class="footer">       
     <div class="content">
-      <p>
-        All rights reserved
-        © 2021
-      </p>
-      <p>
-        <a href="http://www.freepik.com">Graphics by Freepik / macrovector</a> <a href="https://storyset.com/people">/
-          Freepik Storyset</a>
-      </p>
+      <img class="footerLogo" src="assets/whiteLogo.svg"/>           
+      <p class="credits">
+        Qint 2021. All rights reserved        
+      </p>      
+      <navbar>
```

Comment:
There's no such thing as `<navbar>` in strict HTML

Reply:
Changed to div with the class "navbar